helpers/View.php

```diff
@@ -139,33 +159,49 @@ public static function getGlobalJsFileName() {
      * @return string all.css file name
      */
     public static function getGlobalCssFileName() {
-        return 'all-v' . \F3::get('version') . '.css';
+        return self::$staticPrefix . self::$staticmtime['css'] . '.css';
     }
 
     /**
      * generate minified css and js
      *
+     * @param type js or css
+     *
      * @return void
      */
-    public function genMinifiedJsAndCss() {
-        // minify js
-        $targetJs = \F3::get('BASEDIR') . '/public/' . self::getGlobalJsFileName();
-        if (!file_exists($targetJs) || \F3::get('DEBUG') != 0) {
-            $js = '';
-            foreach (\F3::get('js') as $file) {
-                $js = $js . "\n" . $this->minifyJs(file_get_contents(\F3::get('BASEDIR') . '/' . $file));
+    private function genMinified($type) {
+        self::$staticmtime[$type] = self::maxmtime(\F3::get($type));
+
+        if ($type == 'js') {
```

Comment:
Rather than 'js' and 'css', class constants should be created to decrease the chance of accidental mistypes. Since older versions of PHP do no support private constants, `@internal` annotations should be used for now.